DBMS/DBMS3/assignment3.sql

```diff
@@ -0,0 +1,27 @@
+USE library;
+
+
+-- to display total number of different category
+SELECT COUNT(category),category FROM members GROUP BY category;
```

Comment:
ACTION : Requirement was to retrieve the result in single row but  according to you will get multiple rows and that depends on number of categories, Please modify your query to retrieve result in one rows and in that row we will have columns for student count , faculty count etc.